### ISSUE TITLE
[bug] sass loader ?

### ISSUE THREAD

**Issue by ligne13** [roots/bud#1106]
### Agreement

- [X] This is not a duplicate of an existing issue
- [X] I have read the [guidelines for Contributing to Roots Projects](https://github.com/roots/.github/blob/master/CONTRIBUTING.md)
- [X] This is not a personal support request that should be posted on the [Roots Discourse](https://discourse.roots.io/) community

### Expected Behavior

I installed a fresh Bud install with the sass extension, added js and scss entries, and ran `bud build`.

### Actual Behavior

The build throws an error :

```
yarn run v1.22.17
$ /Users/.../roy_visites/node_modules/.bin/bud build
[roy_visites] › ▶  bud          Initialized timer...
[roy_visites] › ❯  process.env.NODE_ENV production
[roy_visites] › ❯  process.env.BABEL_ENV production
▶  building bud Initialized timer...
[roy_visites] › …  [1/5] bootstrap api
[roy_visites] [api] › ❯  binding bud.alias facade
[roy_visites] › ✔  bound alias
[roy_visites] [api] › ✔  binding bud.alias facade
[roy_visites] [api] › ❯  binding bud.assets facade
[roy_visites] › ✔  bound assets
[roy_visites] [api] › ✔  binding bud.assets facade
[roy_visites] [api] › ❯  binding bud.copy facade
[roy_visites] › ✔  bound copy
[roy_visites] [api] › ✔  binding bud.copy facade
[roy_visites] [api] › ❯  binding bud.config facade
[roy_visites] › ✔  bound config
[roy_visites] [api] › ✔  binding bud.config facade
[roy_visites] [api] › ❯  binding bud.webpackConfig facade
[roy_visites] › ✔  bound webpackConfig
[roy_visites] [api] › ✔  binding bud.webpackConfig facade
[roy_visites] [api] › ❯  binding bud.override facade
[roy_visites] › ✔  bound override
[roy_visites] [api] › ✔  binding bud.override facade
[roy_visites] [api] › ❯  binding bud.define facade
[roy_visites] › ✔  bound define
[roy_visites] [api] › ✔  binding bud.define facade
[roy_visites] [api] › ❯  binding bud.devtool facade
[roy_visites] › ✔  bound devtool
[roy_visites] [api] › ✔  binding bud.devtool facade
[roy_visites] [api] › ❯  binding bud.entry facade
[roy_visites] › ✔  bound entry
[roy_visites] [api] › ✔  binding bud.entry facade
[roy_visites] [api] › ❯  binding bud.js facade
[roy_visites] › ✔  bound js
[roy_visites] [api] › ✔  binding bud.js facade
[roy_visites] [api] › ❯  binding bud.css facade
[roy_visites] › ✔  bound css
[roy_visites] [api] › ✔  binding bud.css facade
[roy_visites] [api] › ❯  binding bud.experiments facade
[roy_visites] › ✔  bound experiments
[roy_visites] [api] › ✔  binding bud.experiments facade
[roy_visites] [api] › ❯  binding bud.externals facade
[roy_visites] › ✔  bound externals
[roy_visites] [api] › ✔  binding bud.externals facade
[roy_visites] [api] › ❯  binding bud.hash facade
[roy_visites] › ✔  bound hash
[roy_visites] [api] › ✔  binding bud.hash facade
[roy_visites] [api] › ❯  binding bud.version facade
[roy_visites] › ✔  bound version
[roy_visites] [api] › ✔  binding bud.version facade
[roy_visites] [api] › ❯  binding bud.minimize facade
[roy_visites] › ✔  bound minimize
[roy_visites] [api] › ✔  binding bud.minimize facade
[roy_visites] [api] › ❯  binding bud.persist facade
[roy_visites] › ✔  bound persist
[roy_visites] [api] › ✔  binding bud.persist facade
[roy_visites] [api] › ❯  binding bud.provide facade
[roy_visites] › ✔  bound provide
[roy_visites] [api] › ✔  binding bud.provide facade
[roy_visites] [api] › ❯  binding bud.proxy facade
[roy_visites] › ✔  bound proxy
[roy_visites] [api] › ✔  binding bud.proxy facade
[roy_visites] [api] › ❯  binding bud.run facade
[roy_visites] › ✔  bound run
[roy_visites] [api] › ✔  binding bud.run facade
[roy_visites] [api] › ❯  binding bud.runtime facade
[roy_visites] › ✔  bound runtime
[roy_visites] [api] › ✔  binding bud.runtime facade
[roy_visites] [api] › ❯  binding bud.serve facade
[roy_visites] › ✔  bound serve
[roy_visites] [api] › ✔  binding bud.serve facade
[roy_visites] [api] › ❯  binding bud.splitChunks facade
[roy_visites] › ✔  bound splitChunks
[roy_visites] [api] › ✔  binding bud.splitChunks facade
[roy_visites] [api] › ❯  binding bud.extract facade
[roy_visites] › ✔  bound extract
[roy_visites] [api] › ✔  binding bud.extract facade
[roy_visites] [api] › ❯  binding bud.template facade
[roy_visites] › ✔  bound template
[roy_visites] [api] › ✔  binding bud.template facade
[roy_visites] [api] › ❯  binding bud.use facade
[roy_visites] › ✔  bound use
[roy_visites] [api] › ✔  binding bud.use facade
[roy_visites] [api] › ❯  binding bud.watch facade
[roy_visites] › ✔  bound watch
[roy_visites] [api] › ✔  binding bud.watch facade
[roy_visites] › …  [2/5] bootstrap hooks
[roy_visites] › …  [3/5] bootstrap project
[roy_visites] › …  [4/5] bootstrap env
[roy_visites] [env] › ❯  loading env
[roy_visites] [env] › ⚠  no env values found
[roy_visites] › …  [5/5] bootstrap dashboard
[roy_visites] [dashboard] › ❯  initializing dashboard
[roy_visites] › ✔  [1/5] bootstrap api
[roy_visites] › ✔  [2/5] bootstrap hooks
[roy_visites] › ✔  [4/5] bootstrap env
[roy_visites] [dashboard] › ✔  rendering dashboard
[roy_visites] › ✔  [5/5] bootstrap dashboard
[roy_visites] › ✔  [3/5] bootstrap project
[roy_visites] › …  [1/3] register project
[roy_visites] [env] › ❯  getting publicEnv {}
[roy_visites] › …  [2/3] register build
[roy_visites] › …  [3/3] register dependencies
[roy_visites] › ✔  [3/3] register dependencies
[roy_visites] › ✔  [2/3] register build
[roy_visites] [project] › ▶  building profile Initialized timer...
[roy_visites] [project] › ✖  @roots/bud-postcss manifest cannot be resolved
[roy_visites] [project] › …  reading project configuration files
[roy_visites] [project] › ●  located user config bud.config.js
[roy_visites] [project] › ◼  building profile Timer run for: 76ms
[roy_visites] › ✔  [1/3] register project
[roy_visites] › …  [1/2] registered api
[roy_visites] › …  [2/2] registered build
[roy_visites] › ✔  [2/2] registered build
[roy_visites] › ✔  [1/2] registered api
[roy_visites] › …  [1/3] boot project
[roy_visites] › …  [2/3] boot cache
[roy_visites] › …  [3/3] boot extensions
[roy_visites] [extensions] › ✔  webpack-provide-plugin instantiated
[roy_visites] [extensions] › ✔  clean-webpack-plugin instantiated
[roy_visites] [extensions] › ✔  copy-webpack-plugin instantiated
[roy_visites] [extensions] › ✔  webpack-define-plugin instantiated
[roy_visites] [extensions] › ✔  webpack-hot-module-replacement-plugin instantiated
[roy_visites] [extensions] › ✔  webpack-manifest-plugin instantiated
[roy_visites] [extensions] › ✔  mini-css-extract-plugin instantiated
[roy_visites] › ✔  [1/3] boot project
[roy_visites] [extensions] › ▶  registering  Initialized timer...
[roy_visites] › ❯  registering webpack-provide-plugin
[roy_visites] › ❯  registering clean-webpack-plugin
[roy_visites] › ❯  registering copy-webpack-plugin
[roy_visites] › ❯  registering webpack-define-plugin
[roy_visites] › ❯  registering webpack-hot-module-replacement-plugin
[roy_visites] › ❯  registering webpack-manifest-plugin
[roy_visites] › ❯  registering mini-css-extract-plugin
[roy_visites] [extensions] › ◼  registering  Timer run for: 4ms
[roy_visites] [extensions] › ▶  booting      Initialized timer...
[roy_visites] › ❯  booting webpack-provide-plugin
[roy_visites] › ❯  booting clean-webpack-plugin
[roy_visites] › ❯  booting copy-webpack-plugin
[roy_visites] › ❯  booting webpack-define-plugin
[roy_visites] › ❯  booting webpack-hot-module-replacement-plugin
[roy_visites] › ❯  booting webpack-manifest-plugin
[roy_visites] › ❯  booting mini-css-extract-plugin
[roy_visites] [extensions] › ◼  booting      Timer run for: 2ms
[roy_visites] [extensions] › ✖  missing dependencies in project. not booting extensions.
[roy_visites] › ✔  [3/3] boot extensions
[roy_visites] [cache] › ℹ  cache hash generated xgmmtljsvhpvdrz0kjkp3vw1e6g_
[roy_visites] [cache] › ❯  --cache.type flag filesystem
[roy_visites] [api] › ❯  executing persist ["filesystem"]
[roy_visites] [api] › ✔  cache enabled filesystem
[roy_visites] › ✔  [2/3] boot cache
[roy_visites] › ◼  building bud Timer run for: 652ms
runner ✔  framework ready
▶  process user configs Initialized timer...
❯  processing [secure]/bud.config.js
…  importing module [secure]/bud.config.js
…  calling user config [secure]/bud.config.js
[roy_visites] [api] › ❯  facade called: entry [{"main":["index.js","scss/index.scss"]}]
[roy_visites] [api] › ❯  facade added to queue: entry [{"main":["index.js","scss/index.scss"]}]
[roy_visites] [api] › …  Executing 1 enqueued functions
[roy_visites] [api] › ❯  executing entry [{"main":["index.js","scss/index.scss"]}]
◼  process user configs Timer run for: 3ms
[roy_visites] [api] › ❯  executing run [[]]
[roy_visites] [api] › ❯  event.build.make.promise api calls
[roy_visites] › ℹ  glob search ["index.js","scss/index.scss"]
[roy_visites] › ℹ  glob directory [secure]/src
[roy_visites] › ℹ  glob results ["index.js","scss/index.scss"]
[roy_visites] [api] › ✔  entrypoints added main {"import":["index.js","scss/index.scss"]}
[roy_visites] [extensions] › ▶  extensions.make Initialized timer...
[roy_visites] [extensions] › ❯  webpack-provide-plugin will not be used in the compilation
[roy_visites] [extensions] › ✔  clean-webpack-plugin will be used in the compilation
[roy_visites] [extensions] › ❯  copy-webpack-plugin will not be used in the compilation
[roy_visites] [extensions] › ❯  webpack-define-plugin will not be used in the compilation
[roy_visites] [extensions] › ❯  webpack-hot-module-replacement-plugin will not be used in the compilation
[roy_visites] [extensions] › ✔  webpack-manifest-plugin will be used in the compilation
[roy_visites] [extensions] › ✔  mini-css-extract-plugin will be used in the compilation
[roy_visites] [extensions] › ◼  extensions.make Timer run for: 13ms
[roy_visites] [build] › ⚠  build.make: excluding experiments value is undefined
[roy_visites] [build] › ⚠  build.make: excluding externals value is undefined
[roy_visites] [build] › ⚠  build.make: excluding profile value is undefined
[roy_visites] [build] › ⚠  build.make: excluding watch value is undefined
[roy_visites] [build] › ⚠  build.make: excluding watchOptions value is undefined
[roy_visites] [build] › ❯  writing webpack dump to disk ./.budfiles/bud/webpack.config.js
[roy_visites] [project] › ✔  write profile ./.budfiles/bud/profile.json
[roy_visites] › ℹ  using config from parent compiler
[roy_visites] › ◼  bud          Timer run for: 1.06s
✘ Module parse failed: Unexpected character '@' (1:0)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
> @use 'inherit.scss' as inherit;
| @use '../../template-parts/blocks/audio/style.scss' as audio;
| @use '../../template-parts/blocks/masonry/style.scss' as masonry;

[roy_visites] [dashboard] › …  exiting cli
[roy_visites] › ✔  exiting application
✨  Done in 2.92s.
```

### Steps To Reproduce

1. 
1.
1.

### version

@roots/bud/5.2.0 darwin-x64 node-v16.13.2

### What package manager are you using?

yarn classic

### version

1.22.17

### Are you using pnpm?

- [ ] yes
- [X] no

### Configuration

```shell
module.exports = async bud =>
    bud.entry({main: ['index.js', 'scss/index.scss']})
```

### Relevant .budfiles

profile.json

```
{
  "cache": {
    "hash": "xgmmtljsvhpvdrz0kjkp3vw1e6g_"
  },
  "configs": {
    "dynamic": {
      "global": [
        "/Users/.../roy_visites/bud.config.js"
      ],
      "conditional": []
    },
    "json": {
      "global": [],
      "conditional": []
    }
  },
  "cli": {
    "args": {},
    "argv": [],
    "flags": {},
    "metadata": {},
    "raw": []
  },
  "manifestPath": null,
  "manifest": {
    "name": "roy_visites",
    "private": true,
    "devDependencies": {
      "@roots/bud": "^5.2.0",
      "@roots/bud-sass": "^5.2.0",
      "postcss-scss": "^4.0.3",
      "sass": "^1.45.2",
      "sass-loader": "^12.4.0",
      "stylelint-config-recommended-scss": "^5.0.2"
    },
    "browserslist": "defaults"
  },
  "installed": {
    "@roots/bud": "^5.2.0",
    "@roots/bud-sass": "^5.2.0",
    "postcss-scss": "^4.0.3",
    "sass": "^1.45.2",
    "sass-loader": "^12.4.0",
    "stylelint-config-recommended-scss": "^5.0.2"
  },
  "dependencies": [
    "/Users/.../roy_visites/package.json",
    "/Users/.../roy_visites/bud.config.js"
  ],
  "env": {
    "public": {},
    "all": {}
  },
  "modules": {
    "root": {
      "name": "root",
      "private": true,
      "devDependencies": {
        "@roots/bud": "^5.2.0",
        "@roots/bud-sass": "^5.2.0",
        "postcss-scss": "^4.0.3",
        "sass": "^1.45.2",
        "sass-loader": "^12.4.0",
        "stylelint-config-recommended-scss": "^5.0.2"
      },
      "browserslist": "defaults",
      "version": "0.0.0",
      "bud": null,
      "parent": null,
      "resolvable": true,
      "requires": [
        [
          "@roots/bud",
          "^5.2.0"
        ],
        [
          "@roots/bud-sass",
          "^5.2.0"
        ],
        [
          "postcss-scss",
          "^4.0.3"
        ],
        [
          "sass",
          "^1.45.2"
        ],
        [
          "sass-loader",
          "^12.4.0"
        ],
        [
          "stylelint-config-recommended-scss",
          "^5.0.2"
        ]
      ]
    },
    "@roots/bud-sass": {
      "name": "@roots/bud-sass",
      "version": "5.2.0",
      "bud": {
        "type": "extension",
        "peers": [
          "@roots/bud-postcss"
        ]
      },
      "resolvable": true,
      "peerDependencies": {
        "postcss-scss": "^4.0.3",
        "sass": "^1.45.2",
        "sass-loader": "^12.4.0",
        "stylelint-config-recommended-scss": "^5.0.2"
      },
      "requires": [
        [
          "@roots/bud-postcss",
          "5.2.0"
        ]
      ]
    }
  },
  "adjacents": [
    {
      "name": "@roots/bud-sass",
      "version": "5.2.0",
      "bud": {
        "type": "extension",
        "peers": [
          "@roots/bud-postcss"
        ]
      },
      "resolvable": true,
      "peerDependencies": {
        "postcss-scss": "^4.0.3",
        "sass": "^1.45.2",
        "sass-loader": "^12.4.0",
        "stylelint-config-recommended-scss": "^5.0.2"
      },
      "requires": [
        [
          "@roots/bud-postcss",
          "5.2.0"
        ]
      ]
    }
  ]
}
```

webpack.config.js :

```
module.exports = (Object {
  "bail": true,
  "cache": Object {
    "buildDependencies": Object {
      "bud": Array [
        "/Users/.../roy_visites/package.json",
        "/Users/.../roy_visites/bud.config.js",
      ],
    },
    "cacheDirectory": "/Users/.../roy_visites/.budfiles",
    "managedPaths": Array [
      "/Users/.../roy_visites/node_modules",
    ],
    "type": "filesystem",
    "version": "xgmmtljsvhpvdrz0kjkp3vw1e6g_",
  },
  "context": "/Users/.../roy_visites",
  "devtool": false,
  "entry": Object {
    "main": Object {
      "import": Array [
        "index.js",
        "scss/index.scss",
      ],
    },
  },
  "infrastructureLogging": Object {
    "console": false,
  },
  "mode": "production",
  "module": Object {
    "rules": Array [
      Object {
        "parser": Object {
          "requireEnsure": false,
        },
        "test": /\.[cm]?(jsx?|tsx?)$/,
      },
      Object {
        "oneOf": Array [
          Object {
            "include": "/Users/.../roy_visites/src",
            "test": /\.(js|jsx)$/,
            "use": Array [],
          },
          Object {
            "include": "/Users/.../roy_visites/src",
            "test": /\.css$/,
            "use": Array [
              Object {
                "loader": "/Users/.../roy_visites/node_modules/mini-css-extract-plugin/dist/loader.js",
              },
              Object {
                "loader": "/Users/.../roy_visites/node_modules/css-loader/dist/cjs.js",
                "options": Object {
                  "importLoaders": 1,
                  "sourceMap": false,
                },
              },
            ],
          },
          Object {
            "include": "/Users/.../roy_visites/src",
            "test": /\.module\.css$/,
            "use": Array [
              Object {
                "loader": "/Users/.../roy_visites/node_modules/mini-css-extract-plugin/dist/loader.js",
              },
              Object {
                "loader": "/Users/.../roy_visites/node_modules/css-loader/dist/cjs.js",
                "options": Object {
                  "importLoaders": 1,
                  "localIdentName": "[name]__[local]___[hash:base64:5]",
                  "modules": true,
                  "sourceMap": false,
                },
              },
            ],
          },
          Object {
            "exclude": /(node_modules|bower_components)/,
            "generator": Object {
              "filename": "images/[name][ext]",
            },
            "test": /\.(png|jpe?g|gif)$/,
            "type": "asset/resource",
          },
          Object {
            "generator": Object {
              "filename": "webp/[name][ext]",
            },
            "test": /\.webp$/,
            "type": "asset/resource",
          },
          Object {
            "generator": Object {
              "filename": "svg/[name][ext]",
            },
            "test": /\.svg$/,
            "type": "asset/resource",
          },
          Object {
            "generator": Object {
              "filename": "fonts/[name][ext]",
            },
            "include": "/Users/.../roy_visites/src",
            "test": /\.(ttf|otf|eot|woff2?|ico)$/,
            "type": "asset",
          },
          Object {
            "include": "/Users/.../roy_visites/src",
            "parser": Object {
              "parse": [Function parse],
            },
            "test": /\.json$/,
            "type": "json",
          },
          Object {
            "include": "/Users/.../roy_visites/src",
            "parser": Object {
              "parse": [Function anonymous],
            },
            "test": /\.ya?ml$/,
            "type": "json",
          },
          Object {
            "include": "/Users/.../roy_visites/src",
            "test": /\.(html?)$/,
            "use": Array [
              Object {
                "loader": "/Users/.../roy_visites/node_modules/html-loader/dist/cjs.js",
              },
            ],
          },
          Object {
            "include": "/Users/.../roy_visites/src",
            "test": /\.(csv|tsv)$/,
            "use": Array [
              Object {
                "loader": "/Users/.../roy_visites/node_modules/csv-loader/index.js",
              },
            ],
          },
          Object {
            "include": "/Users/.../roy_visites/src",
            "test": /\.xml$/,
            "use": Array [
              Object {
                "loader": "/Users/.../roy_visites/node_modules/xml-loader/index.js",
              },
            ],
          },
          Object {
            "include": "/Users/.../roy_visites/src",
            "parser": Object {
              "parse": [Function parse],
            },
            "test": /\.(html?)$/,
            "type": "json",
          },
        ],
      },
    ],
  },
  "name": "bud",
  "node": false,
  "optimization": Object {
    "emitOnErrors": false,
    "minimize": false,
    "minimizer": Array [
      "...",
    ],
    "moduleIds": undefined,
    "runtimeChunk": false,
    "splitChunks": false,
  },
  "output": Object {
    "assetModuleFilename": "[name]",
    "chunkFilename": undefined,
    "clean": undefined,
    "filename": "[name].js",
    "path": "/Users/.../roy_visites/dist",
    "pathinfo": undefined,
    "publicPath": "",
  },
  "parallelism": 7,
  "performance": Object {
    "hints": false,
  },
  "plugins": Array [
    CleanWebpackPlugin {
      "apply": [Function bound apply],
      "cleanAfterEveryBuildPatterns": Array [],
      "cleanOnceBeforeBuildPatterns": Array [
        "**/*",
        "!dll",
      ],
      "cleanStaleWebpackAssets": true,
      "currentAssets": Array [],
      "dangerouslyAllowCleanPatternsOutsideProject": false,
      "dry": false,
      "handleDone": [Function bound handleDone],
      "handleInitial": [Function bound handleInitial],
      "initialClean": false,
      "outputPath": "",
      "protectWebpackAssets": true,
      "removeFiles": [Function bound removeFiles],
      "verbose": false,
    },
    WebpackManifestPlugin {
      "options": Object {
        "assetHookStage": Infinity,
        "basePath": "",
        "fileName": "manifest.json",
        "filter": null,
        "generate": undefined,
        "map": null,
        "publicPath": "",
        "removeKeyHash": /([a-f0-9]{16,32}\.?)/gi,
        "seed": undefined,
        "serialize": [Function serialize],
        "sort": null,
        "transformExtensions": /^(gz|map)$/i,
        "useEntryKeys": false,
        "useLegacyEmit": false,
        "writeToFileEmit": true,
      },
    },
    MiniCssExtractPlugin {
      "_sortedModulesCache": WeakMap {},
      "options": Object {
        "chunkFilename": "[name].css",
        "experimentalUseImportModule": undefined,
        "filename": "[name].css",
        "ignoreOrder": false,
        "runtime": true,
      },
      "runtimeOptions": Object {
        "attributes": undefined,
        "insert": undefined,
        "linkType": "text/css",
      },
    },
  ],
  "recordsPath": "/Users/.../roy_visites/.budfiles/bud/modules.json",
  "resolve": Object {
    "alias": Object {},
    "extensions": Array [
      ".wasm",
      ".mjs",
      ".js",
      ".jsx",
      ".css",
      ".json",
      ".json5",
      ".toml",
      ".xml",
      ".csv",
      ".yml",
      ".yaml",
      ".xml",
    ],
    "modules": Array [
      "src",
      "node_modules",
      "/Users/.../roy_visites/node_modules",
    ],
  },
  "stats": Object {
    "assets": true,
    "cached": true,
    "children": true,
    "chunkModules": false,
    "chunks": false,
    "colors": true,
    "entrypoints": false,
    "errors": true,
    "hash": true,
    "modules": false,
    "outputPath": true,
    "performance": false,
    "warnings": true,
  },
  "target": "browserslist:/Users/.../roy_visites/package.json",
})
```

**Comment by kellymears** [comment]
Hey there, sorry the docs are out of date. You don't need to run `bud install` (or `bud init`) after adding extensions as of 5.2.0. 

[See release notes](https://github.com/roots/bud/releases/tag/v5.2.0)

looking at `.budfiles/bud/profile.json`'s `manifest` key:

```json
"manifest": {
    "name": "roy_visites",
    "private": true,
    "devDependencies": {
      "@roots/bud": "^5.2.0",
      "@roots/bud-sass": "^5.2.0",
      "postcss-scss": "^4.0.3",
      "sass": "^1.45.2",
      "sass-loader": "^12.4.0",
      "stylelint-config-recommended-scss": "^5.0.2"
    },
    "browserslist": "defaults"
  },
  ```
  
You should remove `postcss-scss`, `sass`, `sass-loader`, and `stylelint-config-recommended-scss`. These are all provided by the framework as peers. You might consider using the [`@roots/bud-preset-recommend` preset](https://www.npmjs.com/package/@roots/bud-preset-recommend), otherwise I think you'll also need to install `@roots/bud-postcss`, as I believe it is currently a dependency of `@roots/bud-sass`. We won't want to make `@roots/bud-sass` standalone until Webpack fully supports css as a first-class citizen (right now it's an experimental flag).
  
You can import the base stylelint config for `sass` from `@roots/bud-sass/stylelint-config`.

https://github.com/roots/bud/blob/a89a1cf8936e74014846e5038063dfd27bcfd62c/sources/%40roots/bud-sass/stylelint-config/index.js#L1-L6

In the future we plan on supporting running with a custom version of dev dependencies like this, but for now it's best to allow the framework to manage them.

**Comment by aksld** [comment]
Hello,

I have a similiar error :

directors.scss: When linting something other than CSS, you should install an appropriate syntax, e.g. "postcss-scss", and use the "customSyntax" option

**In profile.json :**

```
"devDependencies": {
      "@roots/bud": "5.3.2",
      "@roots/bud-eslint": "5.3.2",
      "@roots/bud-postcss": "5.3.2",
      "@roots/bud-prettier": "5.3.2",
      "@roots/bud-sass": "^5.3.2",
      "@roots/bud-stylelint": "5.3.2",
      "@roots/bud-tailwindcss": "5.3.2",
      "@roots/sage": "5.3.2"
    },
```

**Comment by kellymears** [comment]
@aksld I believe we spoke on slack about this -- it's a stylelint configuration issue. There is a stylelint preset available to configure stylelint for sass (`@roots/bud-sass/stylelint-config`). 

Issues with sass loader were resolved in 5.3.x
